pkg/wdk/primitives.go

```diff
@@ -94,6 +94,10 @@ type PubKeyHex HexString
 
 // Validate checks if the string is valid pubkey hexadecimal string
 func (pkh PubKeyHex) Validate() error {
+	if len(pkh)/2 != PubKeyBytesLenCompressed && len(pkh)/2 != PubKeyBytesLenUncompressed {
```

Comment:
maybe calculate `len(pkh)/2` once 
and maybe add the comment why is this counted like that 🙈